### PR TITLE
docs: add a document about how to build CKB on windows

### PR DESCRIPTION
**Pull request by yangby-cryptape**

### CODE REVIEW

File: docs/ckb-on-windows.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+**All commands should be ran as PowerShell commands.**
+
+### Setup the Build Environment
+
+#### Install Visual Studio 2019
+
+Install [Visual Studio 2019](https://visualstudio.microsoft.com/downloads/)
+with the workload: "Desktop development with C++".
+
+**(minimum required)** Or you can just select two individual components:
```
Comment (doitian) [inline docs/ckb-on-windows.md:14]: 👍

File: docs/ckb-on-windows.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+### Build CKB on Windows 10
+
+- Checkout the source code of [CKB].
+
+  ```posh
+  git clone https://github.com/nervosnetwork/ckb
+  cd ckb
+  ```
+
+- Update the source code of submodules.
```
Comment (doitian) [inline docs/ckb-on-windows.md:56]: We don't need submodules to build CKB

Comment (yangby-cryptape) [reply docs/ckb-on-windows.md]: Resolved!

**Comment by quake** [comment]
bors r=quake,doitian

**Pull request merged**